packages/ui-kit/src/helpers/formatLabels.ts

```diff
@@ -1,9 +1,11 @@
+export type Labels = string[] | string[][]
+
```

Comment:
So wdyt for example:

```suggestion
export type TimeSeriesLabels = string[]
export type LeaderboardLabels = string[][]

type Labels = TimeSeriesLabels | LeaderboardLabels
```

Then we use the exported ones for the props